config.js

```diff
@@ -703,6 +703,8 @@ config.NSFS_TEMP_DIR_NAME = '.noobaa-nsfs';
 
 config.NSFS_FOLDER_OBJECT_NAME = '.folder';
 
+config.NSFS_UNSORTED_OBJECTS_LIST_ENABLED = true; 
```

Comment:
Is this already used by default in this PR? I think this is just confusing right now, so I'd keep it for the next PR improving the list performance, then we can decide how to pick best defaults.